dynamic-layers/virtualization/recipes-containers/docker/docker-moby_git.bbappend

```diff
@@ -10,6 +10,11 @@ do_install:append () {
     install -m 0644 -D ${S}/cli/contrib/completion/bash/docker ${D}/${datadir}/bash-completion/completions/docker
 
     install -m 0644 -D ${WORKDIR}/daemon.json ${D}${sysconfdir}/docker/daemon.json
+
+    sed -i \
+        -e 's/^After=\(.*\)$/After=\1 containerd.service/' \
+        -e 's/^Wants=\(.*\)$/Wants=\1 containerd.service/' \
+        ${D}/${systemd_unitdir}/system/docker.service
```

Comment:
Just a note: in this case it is probably better to use a patch, which will fail e.g. on scarthgap branch.
I'm okay with merging this, but we should remove it, when merging to scarthgap.